src/containers/ReviewProblemStepsModal/components/ReviewProblemStepsContent/hooks.js

```diff
@@ -23,7 +23,12 @@ export const useFeedbackList = (submissionUUID) => {
     if (submissionUUID) {
       setIsLoadingFeedbackList(true);
       try {
-        const data = await api.getFeedbackList(submissionUUID, feedbackListType);
+        let data;
+        if (feedbackListType === 'received') {
+          data = await api.getFeedbackFromList(submissionUUID);
+        } else {
+          data = await api.getFeedbackToList(submissionUUID);
+        }
```

Comment:
Minor suggestion: If we had more options this way I could add more functions 
```
const feedbackListTypesResponses = {
 'received':  api.getFeedbackFromList(submissionUUID), 
 'given': api.getFeedbackToList(submissionUUID), 
};

const response = feedbackListTypes[feedbackListType] || Promise.resolve({ assessments: [] });

const data = await response;

```

Reply:
Thanks for the suggestion! Done